### PR TITLE
enhance creating_a_user_application

### PR DESCRIPTION
**Pull request by couet**
- Add a super simple 3rd case without TApplication
- Change the axis titles of the plots
- Change the picture

May be we should show how to link these 3 programs on windows ?

### CODE REVIEW

**Comment by couet** [comment]
@JohannesHentrich : let me know if these changes are fine for you ?

File: manual/creating_a_user_application/index.md [hunk 1]
```diff
@@ -31,25 +34,70 @@ With the following code a canvas is drawn (→ see also [Graphics]({{ '/manual/g
...
+   g++ demo1.cxx $(root-config --glibs --cflags --libs) -o demo1
+{% endhighlight %}
+
+Then you can run the program as follows:
+
+{% highlight C++ %}
+   ./demo1
+{% endhighlight %}
+
+You will get the following output:
```
Comment (JohannesHentrich) [inline manual/creating_a_user_application/index.md:83]: The following message is displayed:

File: manual/creating_a_user_application/index.md [hunk 2]
```diff
@@ -31,25 +34,70 @@ With the following code a canvas is drawn (→ see also [Graphics]({{ '/manual/g
...
+   TF1 *f1 = new TF1("f1","sin(x)", -5, 5);
+   f1->SetLineColor(kBlue+1);
+   f1->SetTitle("My graph;x; sin(x)");
+   f1->Draw();
+   c->Print("demo1.pdf");
+   return 0;
+}
+{% endhighlight %}
+
+Save the code in file, for example as `demo1.cxx`.
```
Comment (JohannesHentrich) [inline manual/creating_a_user_application/index.md:69]: Save the code in a file, ...

File: manual/creating_a_user_application/index.md [hunk 3]
```diff
@@ -31,25 +34,70 @@ With the following code a canvas is drawn (→ see also [Graphics]({{ '/manual/g
...
+   ./demo1
+{% endhighlight %}
+
+You will get the following output:
+
+{% highlight C++ %}
+Info in <TCanvas::Print>: pdf file demo1.pdf has been created
+{% endhighlight %}
+
+`demo1.pdf` contains the plot of the function `f1`.
```
Comment (JohannesHentrich) [inline manual/creating_a_user_application/index.md:89]: The demo1.pdf file is saved in the user's home directory. The pdf file contains the plot of the `f1`function.

File: manual/creating_a_user_application/index.md [hunk 4]
```diff
@@ -31,25 +34,70 @@ With the following code a canvas is drawn (→ see also [Graphics]({{ '/manual/g
...
 
-The first lines of a header file include ROOT header files. The names of the ROOT header files are almost always the same as the class names (for example, {% include ref class="TApplication" %}, {% include ref class="TF1" %}, {% include ref class="TCanvas" %}).
+## Quitting ROOT when closing the canvas
 
-Use {% include ref class="TApplication" %} to create a ROOT application environment that provides an interface to the windowing system event loops and event handlers. 
 
-To run the canvas as a standalone application you must create a `TApplication` object. Calling the `Run()` method starts the event loop.
+To see the output on screen use {% include ref class="TApplication" %} to create a ROOT application environment that
```
Comment (JohannesHentrich) [inline manual/creating_a_user_application/index.md:94]: Use {% include ref class="TApplication" %} to display the output on a screen. {% include ref class="TApplication" %} creates a ROOT application environment that provides an interface to the windowing system event loops and event handlers.

File: manual/creating_a_user_application/index.md [hunk 5]
```diff
@@ -71,26 +119,28 @@ int main(int argc, char **argv)
 }
 {% endhighlight %}
 
-Save the code in file, for example `demo.cxx`.
+Save the code in file, for example as `demo2.cxx`.
```
Comment (JohannesHentrich) [inline manual/creating_a_user_application/index.md:122]: Save the code in a file,...

File: manual/creating_a_user_application/index.md [hunk 6]
```diff
@@ -110,16 +160,16 @@ int main(int argc, char **argv)
 {% endhighlight %}
 
 
-Save the code in file, for example `demo2.cxx`.
+Save the code in file, for example `demo3.cxx`.
```
Comment (JohannesHentrich) [inline manual/creating_a_user_application/index.md:163]: Save the code in a file...

File: manual/creating_a_user_application/index.md [hunk 7]
```diff
@@ -7,11 +7,14 @@ toc: true
...
-Using the example of creating and displaying a canvas, the following shows how you can create a standalone user application.<br/>
-Two scenarios are presented:
+Using the example of creating and displaying a canvas, the following shows how you can
+create a standalone user application.
 
-- Quitting ROOT when closing the canvas
-- Returning to the ROOT prompt when closing the canvas
+Three scenarios are presented:
+
+- Generating the PDF output of the canvas and quitting.
```
Comment (JohannesHentrich) [inline manual/creating_a_user_application/index.md:15]: may we should link the scenarios with the headings

**Review by JohannesHentrich — COMMENTED** [review]
Only details

File: manual/creating_a_user_application/index.md (hunk 1 above)
Comment (couet) [reply manual/creating_a_user_application/index.md]: done

File: manual/creating_a_user_application/index.md (hunk 5 above)
Comment (couet) [reply manual/creating_a_user_application/index.md]: done

File: manual/creating_a_user_application/index.md (hunk 7 above)
Comment (couet) [reply manual/creating_a_user_application/index.md]: done

File: manual/creating_a_user_application/index.md (hunk 6 above)
Comment (couet) [reply manual/creating_a_user_application/index.md]: done

File: manual/creating_a_user_application/index.md (hunk 3 above)
Comment (couet) [reply manual/creating_a_user_application/index.md]: done

**Pull request merged**